src/app/static/src/app/components/FilterSelect.vue

```diff
@@ -25,24 +26,38 @@
             options: Array,
             value: [Array, String]
         },
+        computed: {
+            treeselectValue() {
+                return this.disabled ? null : this.value;
+            },
+            placeholder() {
+                return this.disabled ? "Not used" : "Select...";
+            }
+        },
         methods: {
             treeselectNormalizer(anyNode: any) {
-                //In the nested case, this gets called for the child nodes we add in below - just return these unchanged
-                if (anyNode.label) {
-                    return anyNode;
-                }
+                const normalize = (anyNode: any) => {
```

Comment:
This was an odd one - adding any computed property which accessed 'this' seemed to make the treeselectNormalizer method inaccessible on this from itself. Not sure why this should be??

Reply:
Yeah I've noticed this too, seems the Vue type inference is a bit off. You can make it work by typing the component more explicitly:
```
    interface Methods {
        treeselectNormalizer: (anyNode: any) => void;
        select: (value: string[]) => void
    }

    interface Computed {
        treeselectValue: string[] | string | null
        placeholder: string
    }

    interface Props {
        multiple: boolean,
        label: string,
        disabled: boolean,
        options: any[],
        value: string[] | string
    }

    export default Vue.extend<{}, Methods, Computed, Props>({
... etc
```